vuestorefront/models/account_move.py

```diff
@@ -0,0 +1,14 @@
+from odoo import models, api
+
+
+class AccountMove(models.Model):
+    _inherit = 'account.move'
+
+    @api.model
+    def prepare_vsf_domain(self, commercial_partner_id, **kwargs):
+        domain = [('message_partner_ids', 'child_of', [commercial_partner_id])]
+        if kwargs.get('ids'):
+            domain.append(('id', 'in', kwargs['ids']))
+        if kwargs.get('line_name'):
+            domain.append(('invoice_line_ids.name', 'ilike', kwargs['line_name']))
+        return domain
```

Comment:
"Line name" should be just addition to the search options that are currently available.

Reply:
@Ernestas-t maybe you can clarify the requirements?